content/en/account_management/rbac/_index.md

```diff
@@ -41,15 +41,13 @@ Datadog Read Only Role
 
 ## Custom roles
 
-<div class="alert alert-warning">
-Creating and modifying custom roles is an opt-in Enterprise feature. <a href="/help">Contact Datadog support</a> to get it enabled for your account.
-</div>
```

Comment:
Once custom roles is GA, this will no longer be the case. There will be a way to enable custom roles directly through the UI.